src/main/scala/centaur/test/Test.scala

```diff
@@ -48,6 +47,9 @@ object Test {
         override def run: Try[A] = Try(x)
       }
     }
+
+    /** Call the default non-stack-safe but correct version of this method. */
+    override def tailRecM[A, B](a: A)(f: (A) => Test[Either[A, B]]): Test[B] = defaultTailRecM(a)(f)
```

Comment:
This should not affect the existing tests.  Cats is just suggesting that the author of a `Monad` instance create a method for stack-safe traversal.  If one declines to do that (as I did here), one can just fall back on the non-stack-safe default implementation.